config/Sage/game_options.txt

```diff
@@ -562,6 +562,9 @@ BOMBCAP 24
 ## of ruins.
 LAVALAND_BUDGET 60
 
+## Space Ruin Budged
+sand_Budget 50
+
```

Comment:
This number is totally random, might be better to make it bigger when we will have a lot of ruins, I'll see in time. And since the Lavaland on is already 60 but whatever